.github/ISSUE_TEMPLATE/a_question.yml

```diff
@@ -0,0 +1,52 @@
+#
+# Copyright (C) 2015 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name: Question 🤔
+description: Usage question or discussion about Fabric8 kubernetes-client
+labels: [kind/question]
```

Comment:
We shouldn't have this either.
We need to encourage users to use [Discussions](https://github.com/fabric8io/kubernetes-client/discussions) instead.

Reply:
So, need to remove entire Question template right?

Reply:
Yes

Reply:
Updated.